audit-resolve.json

```diff
@@ -1,39 +1,39 @@
 {
   "decisions": {
-    "1482|@hapi/hapi": {
-      "decision": "ignore",
-      "madeAt": 1582635042880,
-      "expiresAt": 1583239828928
-    },
     "1179|@mojaloop/central-services-shared>@mojaloop/event-sdk>grpc>node-pre-gyp>mkdirp>minimist": {
```

Comment:
This is a real pain. FYI I did some research on this issue, and it looks like `node-pre-gyp` simply hasn't been updated to the latest version of `mkdirp`, which contains this issue.

Reply:
yeah its a pain, the only thing we can do till its fixed is to ignore the issue, which isn't the best approach